checks/admin_menu.php

```diff
@@ -31,7 +31,7 @@ function check( $php_files, $css_files, $other_files) {
 // Note to TGMPA: Stop trying to bypass theme check. 
 
 		$checks = array(
-			'/(?<!function)[^_>:](add_[^_\'",();]+?_page)/' => _x( 
+			'/(?<!function)[^_>:](add_[^_\'",();][^\s]+?_page)/' => _x( 
```

Comment:
Could this just be simplified to.. ?
```suggestion
			'/(?<!function)[^_>:](add_[a-z]+_page)/' => _x( 
```

Reply:
This code is so old I don't think anyone would remember the reason to why it was originally written this way.